doc/releases/changelog-dev.md

```diff
@@ -211,6 +211,10 @@
 * `qml.transforms.split_non_commuting` will now work with single-term operator arithmetic.
   [(#5314)](https://github.com/PennyLaneAI/pennylane/pull/5314)
 
+* Fixed differentiability for Hamiltonian measurements in new `qutrit_mixed` module. 
+  Tests to confirm measurement and channel differentiation also added
```

Comment:
We don't usually put testing related changes in the changelog

```suggestion
```

Reply:
That makies sense, thank you for catching that!